packages/input-datepicker/src/LionInputDatepicker.js

```diff
@@ -280,17 +290,36 @@ export class LionInputDatepicker extends ScopedElementsMixin(OverlayMixin(LionIn
     `;
   }
 
+  _setupOverlayCtrl() {
+    super._setupOverlayCtrl();
+
+    this.__datepickerBeforeShow = () => {
+      this._overlayCtrl.updateConfig(this._defineOverlayConfig());
+    };
+    this._overlayCtrl.addEventListener('before-show', this.__datepickerBeforeShow);
+  }
+
   /**
    * @override Configures OverlayMixin
    * @desc overrides default configuration options for this component
    * @returns {Object}
    */
   // eslint-disable-next-line class-methods-use-this
   _defineOverlayConfig() {
-    return {
-      ...withModalDialogConfig(),
-      hidesOnOutsideClick: true,
-    };
+    if (window.innerWidth >= 600) {
+      this.hasArrow = true;
+      return {
+        ...withModalDialogConfig(),
+        hidesOnOutsideClick: true,
+        ...super._defineOverlayConfig(),
```

Comment:
This adds placementMode 'local'? https://github.com/ing-bank/lion/blob/master/packages/overlays/src/OverlayMixin.js#L94

Maybe we can make it more explicit by adding `placementMode: 'local'` instead?

Reply:
this actually adds a lot more from the `ArrowMixin`... for placementMode alone yes copying would be better

Reply:
Ah, indeed 👍